examples/index.md

```diff
@@ -162,3 +162,226 @@ Content-Type: application/vnd.api+json
   }
 }
 ```
+
+## Error Objects
+
+Examples of how [error objects](http://jsonapi.org/format/#error-objects) work.
+
+### Responses to failed resource creation:
+
+Response with one error object, with attribute error:
+
+```http
+HTTP/1.1 422 Unprocessable Entity
+Content-Type: application/vnd.api+json
+
+{
+  "errors": [
+    {
+      "title":  "Invalid Attribute",
+      "source": { "pointer": "data/attributes/first-name" },
+      "detail": "First name must contain at least three characters."
+    }
```

Comment:
ref in the wild: https://github.com/cerebris/jsonapi-resources/blob/28ac3971f8b9e73073575d41439ffe263d7e56f5/test/controllers/controller_test.rb#L2412-L2429